react/components/UserWidget.tsx

```diff
@@ -326,7 +326,7 @@ const UserWidget: VtexFunctionComponent<UserWidgetProps> = ({
       return
     }
 
-    const uiSettings = userWidgetData?.getB2BSettings?.uiSettings
+    const uiSettings = userWidgetData?.getB2BSettings
```

Comment:
I'm not sure this is correct, removing the `uiSettings` here. Was it broken before?